onadata/libs/utils/api_export_tools.py

```diff
@@ -638,9 +641,11 @@ def _get_google_credential(request):
         credential = Credentials(token=request.session["access_token"])
 
     if credential and not credential.valid:
-        # We need the user to provide a new crential
-        storage.delete()
-        credential = None
+        try:
+            credential.refresh(Request())
```

Comment:
Should we store the credential again once we've refreshed it ? _We may also need to somehow block other requests from refreshing the token while we refresh it in the future perhaps..._